docker/images/python/Dockerfile.dev

```diff
@@ -0,0 +1,15 @@
+FROM python:slim
+
+RUN apt-get update \
+    # wget is needed for working with ftp
+    # git is needed for pip
+    && apt-get install --no-install-recommends --no-install-suggests -y \
+    wget git make wkhtmltopdf xvfb gettext
```

Comment:
seems it will be enough`wkhtmltopdf xvfb gettext`

Reply:
`git` and `make` are required too